packages/compiler-cli/test/ngtsc/ngtsc_spec.ts

```diff
@@ -4369,6 +4369,54 @@ runInEachFileSystem(os => {
          expect(jsContents).toMatch(setClassMetadataRegExp('type: undefined'));
        });
 
+    it('should use `undefined` in setClassMetadata if enums can\'t be represented as values',
```

Comment:
nit: I'd personally mention "const enums" here explicitly.

```suggestion
    it('should use `undefined` in setClassMetadata for const enums',
```